src/features/device-state/state-patch-utils.ts

```diff
@@ -39,6 +39,44 @@ export const v2ValidPatchFields: Array<
 	'download_progress',
 ];
 
+const SHORT_TEXT_LENGTH = 255;
+const ADDRESS_DELIMITER = ' ';
+
+// Truncate space-delimited addresses at the space character to SHORT_TEXT_LENGTH
+const truncateIpMacAddresses = (address: string): string => {
+	return address
+		.split(ADDRESS_DELIMITER)
+		.reduce((addresses, addr) => {
+			return addresses.length + ADDRESS_DELIMITER.length + addr.length <
```

Comment:
Can we not use the full SHORT_TEXT_LENGTH here? IOW, why `<` and not `<=` ?

I'd worked on the support ticket that led to creation of this issue hence checking out the fix. Hope I'm not missing something obvious here.

Reply:
No specific reason to not use the full length of SHORT_TEXT_LENGTH

Reply:
If the current address string length is equal to (`<=`) SHORT_TEXT_LENGTH, it would add the next address in (`addresses + ADDRESS_DELIMITER + addr`), making the total length greater than 255. 

Also, if we do something like `return address.length < SHORT_TEXT_LENGTH ? addresses + DELIMITER + addr` (this is what I assume is meant by using the full SHORT_TEXT_LENGTH), it may be that `addresses + DELIMITER + addr` will be greater than 255 chars already. Thus we need to have the `addresses.length + ADDRESS_DELIMITER.length + addr.length` lookahead logic to make sure we never pass 255 chars.

Okay with you @pranavpeshwe @fisehara, or is there something I'm missing?

Reply:
@cywang117 Thanks for replying. I did not get the explanation, though.
What I meant to ask is, could it not have been ` return addresses.length + ADDRESS_DELIMITER.length + addr.length <= SHORT_TEXT_LENGTH  ? addresses + ADDRESS_DELIMITER + addr : addresses;` ?  Why are we limiting ourselves to only 254 characters out of the available 255 by using `<` instead of `<=` in the check?

Reply:
Ahh I see.. I think you're right and it should be `<=`. Changing it with the next force push